pulp_2to3_migration/app/pre_migration.py

```diff
@@ -480,3 +480,14 @@
                     pub_model.objects.all().delete()
                 for distribution_model in distributor_migrator.pulp3_distribution_models:
                     distribution_model.objects.all().delete()
+            for pulp2_repo in Pulp2Repository.objects.filter(type=plugin.type, is_migrated=True):
+                if pulp2_repo.pulp3_repository_version:
```

Comment:
So, this wipes out all the Pulp 3 Repositories and RepositoryVersions for pre-migrated repos with ```is_migrated```.  Probably it could be done simpler if we just wipe all the Repositories of the correct type?